webapp/components/get_link_modal.jsx

```diff
@@ -28,6 +29,13 @@ export default class GetLinkModal extends React.Component {
         this.props.onHide();
     }
 
+    copyLinkOnClick() {
+        $(ReactDOM.findDOMNode(this.refs.textarea)).on('click', function copyLinkOnClick() {
```

Comment:
Few things:
1. If the ref is an HTML element you don't need `ReactDOM.findDOMNode()` you can just access it directly
2. This event listener never gets removed, can you remove it in `componntWillUnmount()`?
3. This event listener looks like a bit of a hack, what's the purpose of it?